tests/Behat/features/preview-a-version.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Preview a selected version
+  As a cms author
+  I want to preview a selected version
+
+  // Given a preview panel exists for this page
```

Comment:
This feature looks incomplete - remove?

Reply:
Great that you caught that. It is WIP indeed and I only accidentally added it to the PR.